config/conf.toml

```diff
@@ -2,3 +2,11 @@
 tsdb.backend = 'marketstore'
 tsdb.host = 'localhost'
 tsdb.grpc_port = 5995
+
+[ui]
+# set custom font + size which will scale entire UI
+# font_size = 16
+# font_name = 'Monospaced'
+
+# colorscheme = 'default'  # UNUSED
```

Comment:
these are just "wishlist" stuff rn since nothing implemented but figured i might as well toss em in for a keen piker to take a swing at 😉